### PR TITLE
Made HW19

### PR DESCRIPTION
**Pull request by anastasiya-harbatsevich**
Create "UserController" and use it in tests. 
Create "CartController" and add "CartTest"
Improve "Product ID"

### CODE REVIEW

File: src/main/java/com/practicesoftwaretesting/cart/model/Product.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Product{
```
Comment (alexshamrai) [inline src/main/java/com/practicesoftwaretesting/cart/model/Product.java]: format your code, please
```suggestion
public class Product {
```

File: src/main/java/com/practicesoftwaretesting/cart/model/CartItemsInfoResponse.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.practicesoftwaretesting.cart.model;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CartItemsInfoResponse{
```
Comment (alexshamrai) [inline src/main/java/com/practicesoftwaretesting/cart/model/CartItemsInfoResponse.java]: Format your code
```suggestion
public class CartItemsInfoResponse {
```

File: src/main/java/com/practicesoftwaretesting/product/model/Brand.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.practicesoftwaretesting.product.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Brand{
```
Comment (alexshamrai) [inline src/main/java/com/practicesoftwaretesting/product/model/Brand.java]: the same

**Review by alexshamrai — APPROVED** [review]

File: src/main/java/com/practicesoftwaretesting/cart/model/Product.java (hunk 1 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/com/practicesoftwaretesting/cart/model/Product.java]: Formatted

File: src/main/java/com/practicesoftwaretesting/product/model/Brand.java (hunk 3 above)
Comment (anastasiya-harbatsevich) [reply src/main/java/com/practicesoftwaretesting/product/model/Brand.java]: Formatted

**Pull request merged**